holidays/registry.py

```diff
@@ -177,6 +178,8 @@
     "ny_stock_exchange": ("NewYorkStockExchange", "NYSE", "XNYS"),
 }
 
+IMPORT_ENSURE_EXECUTOR = ThreadPoolExecutor(max_workers=1, thread_name_prefix="[HoliDynImp] ")
```

Comment:
In linux, as well as other *nix systems, thread names have a pretty show maximum length, in linux it's[ 16-bytes (including trailing \0)](https://man7.org/linux/man-pages/man3/pthread_getname_np.3.html#:~:text=whose%20length%20is%20restricted%20to%2016%0A%20%20%20%20%20%20%20characters%2C%20including%20the%20terminating%20null%20byte%20(%27%5C0%27)).  So i've abbreviated to 13 bytes in the prefix, which is already rather long.  and will only allow for the first two bytes of the python binary name `py` to show up.  Feel free to suggest a different abbreviation.